pkg/virt-handler/vm_test.go

```diff
@@ -794,139 +794,130 @@ var _ = Describe("VirtualMachineInstance", func() {
-			// should not make another event entry unless something changes
-			expectEvent(string(v1.AccessCredentialsSyncSuccess), false)
-		})
+				sanityExecute()
+				// should not make another event entry unless something changes
+				expectEvent(string(v1.AccessCredentialsSyncSuccess), false)
+			})
 
-		It("should update access credential condition if agent disconnects", func() {
-			vmi := api2.NewMinimalVMI("testvmi")
-			vmi.UID = vmiTestUUID
-			vmi.ObjectMeta.ResourceVersion = "1"
-			vmi.Status.Phase = v1.Running
-			vmi = addActivePods(vmi, podTestUUID, host)
-			vmi.Status.Conditions = []v1.VirtualMachineInstanceCondition{
-				{
+			It("should update condition if agent disconnects", func() {
+				vmi.Status.Conditions = []v1.VirtualMachineInstanceCondition{{
 					Type:          v1.VirtualMachineInstanceAccessCredentialsSynchronized,
 					LastProbeTime: metav1.Now(),
 					Status:        k8sv1.ConditionTrue,
-				},
-				{
-					Type:   v1.VirtualMachineInstanceIsMigratable,
-					Status: k8sv1.ConditionTrue,
-				},
-			}
+				}}
 
-			domain := api.NewMinimalDomainWithUUID("testvmi", vmiTestUUID)
-			domain.Status.Status = api.Running
-			domain.Spec.Metadata.KubeVirt.AccessCredential = &api.AccessCredentialMetadata{
-				Succeeded: false,
-				Message:   "some message",
-			}
-
-			vmiFeeder.Add(vmi)
-			domainFeeder.Add(domain)
-			createVMI(vmi)
+				domain.Spec.Metadata.KubeVirt.AccessCredential = &api.AccessCredentialMetadata{
+					Succeeded: false,
+					Message:   "some message",
+				}
 
-			client.EXPECT().SyncVirtualMachine(vmi, gomock.Any())
-			mockHotplugVolumeMounter.EXPECT().Unmount(gomock.Any(), mockCgroupManager).Return(nil)
-			mockHotplugVolumeMounter.EXPECT().Mount(gomock.Any(), mockCgroupManager).Return(nil)
+				prepare()
 
-			sanityExecute()
+				sanityExecute()
 
-			expectEvent(string(v1.AccessCredentialsSyncFailed), true)
-			updatedVMI, err := virtfakeClient.KubevirtV1().VirtualMachineInstances(metav1.NamespaceDefault).Get(context.TODO(), vmi.Name, metav1.GetOptions{})
-			Expect(err).NotTo(HaveOccurred())
-			Expect(updatedVMI.Status.Conditions).To(ConsistOf(
-				MatchFields(IgnoreExtras, Fields{
-					"Type":   Equal(v1.VirtualMachineInstanceIsMigratable),
-					"Status": Equal(k8sv1.ConditionTrue)},
-				),
-				MatchFields(IgnoreExtras, Fields{
-					"Type":    Equal(v1.VirtualMachineInstanceAccessCredentialsSynchronized),
-					"Status":  Equal(k8sv1.ConditionFalse),
-					"Message": Equal("some message")},
-				),
-				MatchFields(IgnoreExtras, Fields{
-					"Type":   Equal(v1.VirtualMachineInstanceIsStorageLiveMigratable),
-					"Status": Equal(k8sv1.ConditionTrue)},
-				),
-			))
+				expectEvent(string(v1.AccessCredentialsSyncFailed), true)
+				updatedVMI, err := virtfakeClient.KubevirtV1().VirtualMachineInstances(metav1.NamespaceDefault).Get(context.TODO(), vmi.Name, metav1.GetOptions{})
+				Expect(err).NotTo(HaveOccurred())
+				Expect(updatedVMI.Status.Conditions).To(ContainElement(
```

Comment:
Use `matcher.HaveConditionFalse` if you don't care about the message.

Reply:
done